### DISCUSSION TITLE
Pushing a branch

### DISCUSSION THREAD

**Discussion by sannesofie** [lab-antwerp-1/home#5] [Q&A]
I have been trying to push a branch all morning. I have a repository with my bio and I would love to somehow add that to the group repository... but I am currently lost in that

**Comment by denrique-alvarez** [comment]
Hi Sanne! How are you trying to do it? From GIT Desktop maybe?

**Reply by sannesofie** [reply]
Hi Daniel, I am doing it (while watching "Branches - Git and Github or Poets" but when I am in my groups repository I do not get the option to create a new branch like he does in his video ( maybe I did not get rights to do that) 🤔

**Reply by denrique-alvarez** [reply]
I think you might be right. You can ask @edualfaro if you have "writing access" to it. I was checking and apparently, you did send a PR and even forked the repository. I think checking the permission and doing it with GitHub desktop would help. I find it a little bit more intuitive. Also when you are cloning, which is the best option in this case.

**Reply by sannesofie** [reply]
I tried to make the branch from my profile and that worked now tried to PR it into the master but I get this messag
<img width="1438" alt="Screenshot 2021-10-04 at 13 18 33" src="https://user-images.githubusercontent.com/87134682/135842798-e794ca45-eaeb-4002-b1de-65194cac56e7.png">
e

**Reply by denrique-alvarez** [reply]
Forget what I said before. I got the exact same error message. I don't know exactly what it is. However, the PR was registered successfully in my case. How about you?

**Reply by sannesofie** [reply]
I tried to do the markdown in VS code to make sure lint was followed, I then copy pasted into a new PR but still same message. I am not yet strong in either VS code or Terminal so have no idea what I am doing when in there

**Reply by denrique-alvarez** [reply]
Did you follow the steps of the README.md file? I mean, did you run the commands of the 'Getting started' and 'Code quality checks' sections in your terminal of VsCode?

**Reply by sannesofie** [reply]
Hi Daniel, no I did not (still a bit afraid of Terminal) 🙈 Looking for the README? Do you have a link?

**Reply by denrique-alvarez** [reply]
Hi Sanne. Sorry for the late reply. [Here is the link](https://github.com/lab-antwerp-1/fantastic-four#readme)

**Comment by sannesofie** [comment]
Latest development. Can I press yes?
<img width="567" alt="Screenshot 2021-10-04 at 19 22 03" src="https://user-images.githubusercontent.com/87134682/135896176-0b5cc7a3-6cd2-496a-853c-4d5a92da781d.png">

**Reply by denrique-alvarez** [reply]
Were you able to get it right? I have not been able to pass all the checks despite running the three commands to control lint, format and spelling in my terminal.
![Format commands](https://user-images.githubusercontent.com/90314185/136003302-57c8eff1-311c-4555-add4-b398db06ed77.png)
![Error](https://user-images.githubusercontent.com/90314185/136003319-4f8b5437-ad5a-489f-9567-54a7a4581395.png)

**Reply by sannesofie** [reply]
Hi Daniel, no I am still struggling with this problem...

**Reply by denrique-alvarez** [reply]
Hi @gelilaa @colevandersWands. Could you please check the screenshots above and tell me if I am missing something? Thanks in advance!

**Comment by edualfaro** [comment]
Good morning team,

@sannesofie totally my fault, I did not give to each member of our group write access to the group repo, now it is done!, all members of the **[fantastic four](https://github.com/lab-antwerp-1/fantastic-four)** group are admins with full access and privileges.

So sorry for the "frustration moment" because this issue.

![image](https://user-images.githubusercontent.com/87333985/135972700-cf072ef3-ee30-46c3-a5d6-8dc008711f50.png)

**Comment by sannesofie** [comment]
Now my next struggle is that for some reason I am not allowed to make a pull request in lab-antwerp-1/home. I tried forking it and adding my vocabulary file and making a pull request on my own profile, no problem, but I can do anything on the lab-antwerp-1 page 👎 #helpneeded @gelilaa or @colevandersWands

**Reply by gelilaa** [reply]
@sannesofie it seems like you are trying to push directly to the home repository.  instead of using the command `git push`
use `git push origin master`

to be sure can you please share your screen shot here?

**Reply by sannesofie** [reply]
Hi Gelila, I shared a screen recording in Slack as it was too big for here. I am still quite intimidated by Terminal, I watched 20 tutorials yesterday and still don't feel very secure in using Terminal. As you can see in previous screen shots I am not even sure it is setup correctly 🙈 any ideas are welcome as to best proceed
